index.d.ts

```diff
@@ -1,12 +1,6 @@
 import { Plugin } from 'webpack'
 
 interface IMonacoEditorWebpackPluginOpts {
-    /**
-     * custom output path for worker scripts, relative to the main webpack `output.path`.
-     * Defaults to ''.
-     */
-    output?: string;
```

Comment:
This new option subsumes the old `output` option since filenames can include directories through use of a slash.